getting-started/admin.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Administrators
+description: Getting started with Coder as an administrator.
+---
+
+This article will walk you through the steps needed to set up, deploy, and
+configure Coder so that your developers are ready create their workspaces and
+begin working on their projects.
+
+## Set up and deploy Coder
+
+To get started with Coder, you’ll need to [deploy Coder](./setup/index.md) to a
+Kubernetes cluster. We have documentation to help you
+[create a cluster](./setup/kubernetes/index.md) if needed. Once you have a
+cluster, you can [install Coder via Helm](./setup/installation.md).
+
+Alternatively, you can leverage [Coder for Docker](../setup/docker.md), which
```

Comment:
seems a little odd that some most links refer to `./setup` and this one is using `../setup`